src/app.js

```diff
@@ -151,6 +153,8 @@ class Application {
             `An error occured while converting JSON summary to HTML report.`,
             error
           );
+          logger.error(error?.stack);
```

Comment:
I would update the logger.error method so that the stack in printed

Reply:
done